consensus/src/runway/mod.rs

```diff
@@ -640,6 +667,19 @@ where
                     }
                 },
 
+                message = units_from_backup => match message {
+                    Ok(units) => {
+                        for u in units {
+                            self.save_unit(u.clone());
```

Comment:
Well, we thought about how units should be backed up and w came up with three solutions:
1. We have one backup, and pass a clone of that to `UnitLoader`. `UnitSaver` should append this backup with newly created units.
2. We have one backup per run. `UnitLoader` should have all of those backups concatenated. `UnitSaver` should write to a new backup newly created units.
3. We have one backup per run. `UnitLoader` should have the latest backup. `UnitSaver` should write to a new backup all units, together with the ones loaded.
The solution implemented is the 3rd solution. From perspective of api first two solutions are the same. When I now think about it maybe we should go with option number 2? Solution number 3 makes room for an error if node crashes during saving loaded backup. Also this way we unnecessarily save units more than once.

Reply:
Because of this unlikely scenario that we crash during saving the units from backup I would think 1, 2 are a little better than 3. Out of these 3 I like 1st the most. 

The only change in AlephBFT between 1,2 vs 3 is this line, right? I.e., whether we save the loaded ones here as well, or not.

Reply:
Yes, the only difference is this line. 

Timorl had some objections against number 1, as it is generally harder to append a file then to save to a new one.

Reply:
Hmm, my concern is mostly that saving units one by one here (and flushing every time) might be quite slow, and by that I mean REALLY slow -- can't be sure but this might be of the order of 1s even for writing 1500 units (which is a number that can happen). And the consequence might be that when this is interrupted at some point then either we end up holding a backup file with a subset of units written, or a corrupted file.

But that's just a concern to consider -- in general I'm ok with the design :)

Reply:
I changed it to append. I also modified the tests so that they fail, if AlephBFT would change save mechanism from append to new file and save all.